### PR TITLE
Update airbus logo

### LINKED ISSUE #10420: Airbus icon is missing brand guidelines
### Brand Name

Airbus

### Official Resources for Icon and Color

https://brand.airbus.com/en/asset-library/airbus-logo

### Additional Comments

The brand guidelines are missing from this icon. There are also multiple options for the color but the blue one is most common.

### PR DESCRIPTION
**Pull request by Garbg**
Airbus

**Issue:** closes #10420, contributes to #10282

**Similarweb rank:**
Doesn't matter, updating an existing Icon

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I didn't manually vectorize the logo, I got it from the navbar of the airbus website.

### CODE REVIEW

File: icons/airbus.svg [hunk 1]
```diff
@@ -1 +1 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>Airbus</title><path d="M11.067 11.296c0-.815-.53-1.433-1.63-1.433H7.226v4.262h1.037V10.79h1.174c.432 0 .593.247.593.53 0 .297-.16.532-.593.532h-1l1.432 2.273h1.186s-.976-1.532-.963-1.532c.592-.136.975-.556.975-1.297m-5.57-1.433h1.037v4.262H5.497Zm-3.323 0L0 14.137h1.173l.346-.717h1.754l-.432-.901h-.877l.642-1.31h.013l1.457 2.928h1.198L3.1 9.863Zm12.685 2.05c.334-.185.507-.469.507-.901 0-.692-.519-1.149-1.371-1.149H11.56v4.262h2.52c.877 0 1.458-.47 1.458-1.199a1.016 1.016 0 0 0-.68-1.012m-2.26-1.15h1.383c.21 0 .37.161.37.371s-.16.37-.37.37H12.6zm1.42 2.471H12.6v-.864h1.42a.433.433 0 1 1 0 .864m4.805-.988c0 .642-.296 1-.889 1-.58 0-.877-.358-.877-1V9.863h-1.062v2.31c0 1.322.691 2.05 1.951 2.05 1.26 0 1.952-.728 1.952-2.05v-2.31h-1.062v2.384zm3.842-.68c-.988-.234-1.198-.259-1.198-.543 0-.222.247-.32.667-.32.555 0 1.136.135 1.47.345l.333-.865c-.445-.247-1.062-.407-1.803-.407-1.1 0-1.717.543-1.717 1.284 0 .79.457 1.112 1.519 1.334.828.186 1 .297 1 .531 0 .247-.222.359-.679.359-.655 0-1.235-.161-1.704-.42l-.322.914c.52.271 1.285.444 2.05.444 1.075 0 1.718-.494 1.718-1.334.012-.679-.42-1.111-1.334-1.321"/></svg>
```
Comment (adamrusted) [inline icons/airbus.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Airbus</title><path d="M11.067 11.296c0-.815-.53-1.433-1.63-1.433H7.226v4.262h1.037V10.79h1.174c.432 0 .593.247.593.53 0 .297-.16.532-.593.532h-1l1.432 2.273h1.186s-.976-1.532-.963-1.532c.592-.136.975-.556.975-1.297m-5.57-1.433h1.037v4.262H5.497Zm-3.323 0L0 14.137h1.173l.346-.717h1.754l-.432-.901h-.877l.642-1.31h.013l1.457 2.928h1.198L3.1 9.863Zm12.685 2.05c.334-.185.507-.469.507-.901 0-.692-.519-1.149-1.371-1.149H11.56v4.262h2.52c.877 0 1.458-.47 1.458-1.199a1.016 1.016 0 0 0-.68-1.012m-2.26-1.15h1.383c.21 0 .37.161.37.371s-.16.37-.37.37H12.6zm1.42 2.471H12.6v-.864h1.42a.433.433 0 1 1 0 .864m4.805-.988c0 .642-.296 1-.889 1-.58 0-.877-.358-.877-1V9.863h-1.062v2.31c0 1.322.691 2.05 1.951 2.05 1.26 0 1.952-.728 1.952-2.05v-2.31h-1.062v2.384zm3.842-.68c-.988-.234-1.198-.259-1.198-.543 0-.222.247-.32.667-.32.555 0 1.136.135 1.47.345l.333-.865c-.445-.247-1.062-.407-1.803-.407-1.1 0-1.717.543-1.717 1.284 0 .79.457 1.112 1.519 1.334.828.186 1 .297 1 .531 0 .247-.222.359-.679.359-.655 0-1.235-.161-1.704-.42l-.322.914c.52.271 1.285.444 2.05.444 1.075 0 1.718-.494 1.718-1.334.012-.679-.42-1.111-1.334-1.321"/></svg>
```

Need to add the `role` attribute, and re-order them for our linter to pass!

**Comment by Garbg** [comment]
New accenture logo:
accenture

**Comment by Garbg** [comment]
Got it from the top of their website, no changes to _data/simple_icons.json were needed as previous info was correct

**Comment by Garbg** [comment]
I meant to have it on a different pull request but I forgot to do it on a seperate branch

**Comment by Garbg** [comment]
Oopsie meant to have the accenture logo update on a different branch

**Comment by PeterShaggyNoble** [comment]
@Garbg, you can remove it from this branch and then create a new branch & PR for it.

**Review by PeterShaggyNoble — APPROVED** [review]
Perfect, @Garbg, nice work 👍🏻

Note for other @simple-icons/maintainers: This update does not change the fact that we may need to remove this icon as part of the [Great Icon Review](https://github.com/simple-icons/simple-icons/issues/10282) as noted by @adamrusted.

**Pull request merged**